README.md

```diff
@@ -141,6 +141,7 @@ Flags:
       --mock-trigger-time string   mock trigger time in RFC3339 format. This takes priority over mock-delay-sec (default: none)
   -p, --port string                the HTTP port where the mock runs (default: 1338)
   -s, --save-config-to-file        whether to save processed config from all input sources in .ec2-metadata-mock/.aemm-config-used.json in $HOME or working dir, if homedir is not found (default: false)
+  -x, --termination-nodes int      number of nodes in a cluster that can receive Spot interrupt notice (default: 99, meaning unlimited nodes) (default 99)
```

Comment:
Should this apply to scheduled maintenance events as well with a more generic name?

Reply:
Yeah, it definitely should. Not that I'm hyped about `--termination-nodes` as a name, but could apply to both if i update the description to be non-spot specific. 

Thoughts on keeping `-x` as shorthand?